packages/java/package.json

```diff
@@ -54,5 +55,10 @@
   "nyc": {
     "extends": "../../configs/nyc.json"
   },
-  "jdt.ls.download.path": "/jdtls/milestones/0.18.0/jdt-language-server-0.18.0-201805010001.tar.gz"
+  "ls": {
+    "version": "0.0.1",
```

Comment:
how is retrieved this version (not the dev version) ?

Reply:
That's just bogus, I'll rewrite it.